cmd/kubelet/app/server.go

```diff
@@ -553,7 +553,7 @@ func createClientConfig(s *options.KubeletServer) (*restclient.Config, error) {
 		if err != nil {
 			return nil, err
 		}
-		authConfig.Host = s.APIServerList[0]
```

Comment:
this looks like it could leave `Host` empty. `AlternateHosts` is an odd field name if the primary `Host` field can be unset.